internal/support/logging/router.go

```diff
@@ -32,8 +34,8 @@ func pingHandler(w http.ResponseWriter, _ *http.Request) {
 	w.Write([]byte("pong"))
 }
 
-func configHandler(w http.ResponseWriter, _ *http.Request) {
-	pkg.Encode(Configuration, w, LoggingClient)
+func configHandler(w http.ResponseWriter, _ *http.Request, loggingClient logger.LoggingClient) {
```

Comment:
Called out once; occurs elsewhere.

Removed unused parameters from the handler signature.  With the creation of a closure in the route's definition, the unused parameters are no longer needed to fulfill HandleFunc's handler func(ResponseWriter, *Request) signature.